backend/tests/e2e/conftest.py

```
@@ -0,0 +1,471 @@
+            try:
+                if os.path.exists(temp_file):
+                    os.remove(temp_file)
+            except (OSError, IOError):
+                pass  # Ignore cleanup errors
+        
+        # Clean up temporary directory
+        if self.temp_dir and self.temp_dir.exists():
+            try:
+                shutil.rmtree(self.temp_dir)
+            except (OSError, IOError):
+                pass  # Ignore cleanup errors
```

Ruff (0.12.2)

Use `contextlib.suppress(OSError, IOError)` instead of `try`-`except`-`pass`

(SIM105)